.circleci/config.yml

```diff
@@ -0,0 +1,93 @@
+version: 2
+jobs:
+  test:
+    docker:
+      - image: python:2
+        environment:
+          CKAN_DATASTORE_POSTGRES_DB: datastore_test
+          CKAN_DATASTORE_POSTGRES_READ_USER: datastore_read
+          CKAN_DATASTORE_POSTGRES_READ_PWD: pass
+          CKAN_DATASTORE_POSTGRES_WRITE_USER: datastore_write
+          CKAN_DATASTORE_POSTGRES_WRITE_PWD: pass
+          CKAN_POSTGRES_DB: ckan_test
+          CKAN_POSTGRES_USER: ckan_default
+          CKAN_POSTGRES_PWD: pass
+          PGPASSWORD: ckan
+          NODE_TESTS_CONTAINER: 2
+      - image: postgres:10
+        environment:
+          POSTGRES_USER: ckan
+          POSTGRES_PASSWORD: ckan
+        name: ckan-postgres
+      - image: redis:3
+        name: ckan-redis
+      - image: ckan/solr:latest
+        name: ckan-solr
```

Comment:
This doesn't use the solr configuration from the code being tested. We'll need to copy the schema over if we use to the docker image like this.